### PR TITLE
Bug 2091542: - Error Loading/404 not found page shown after clicking "Current namespace only"

### PR DESCRIPTION
**Pull request by cyril-ui-developer**
Also, this PR addressed [Namespace value is missing on the list when selecting "All namespaces" for operators ](https://bugzilla.redhat.com/show_bug.cgi?id=2091545) bug

### CODE REVIEW

File: frontend/packages/operator-lifecycle-manager/src/components/operand/index.tsx [hunk 1]
```diff
@@ -942,6 +941,7 @@ type Header = {
 
 export type OperandTableRowProps = RowFunctionArgs<K8sResourceKind> & {
   csv?: ClusterServiceVersionKind;
```
Comment (TheRealJon) [inline frontend/packages/operator-lifecycle-manager/src/components/operand/index.tsx:943]:
```suggestion
```

**Review by TheRealJon — CHANGES_REQUESTED** [review]
We should remove `csv` from OperandListProps and OperandTableRowProps types since they are no longer used in those components.

File: frontend/packages/operator-lifecycle-manager/src/components/operand/index.tsx [hunk 2]
```diff
@@ -867,6 +865,7 @@ export type OperandListProps = {
...
```
[the remark this answers is not in the document]
Comment (cyril-ui-developer) [reply frontend/packages/operator-lifecycle-manager/src/components/operand/index.tsx]: > 

@TheRealJon Changed this to `obj` .

[the remark this answers is not in the document]
Comment (cyril-ui-developer) [reply frontend/packages/operator-lifecycle-manager/src/components/operand/index.tsx]: > > 
> 
> @TheRealJon Changed this to `obj` .

Never mind. Remove it.

**Review by TheRealJon — APPROVED** [review]
/lgtm

**Comment by TheRealJon** [comment]
@cyril-ui-developer For future reference, if a PR addresses multiple bugs, you can add all the bugs in the title as a comma-separated list like this: "Bug 1234567, Bug 1234568: Fixes two bugs".

**Comment by cyril-ui-developer** [comment]
> @cyril-ui-developer For future reference, if a PR addresses multiple bugs, you can add all the bugs in the title as a comma-separated list like this: "Bug 1234567, Bug 1234568: Fixes two bugs".

Thanks. 
@TheRealJon This needs to be backported to the release branch, right?

**Pull request merged**